packages/bpk-component-map/src/BpkPriceMarkerV2/BpkPriceMarker.module.scss

```diff
@@ -0,0 +1,62 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../../bpk-mixins/index.scss';
+
+.bpk-price-marker {
+  display: flex;
+  height: bpk-spacing-lg();
+  padding: 0 bpk-spacing-md();
+  justify-content: center;
+  align-items: center;
+  border-radius: $bpk-border-radius-sm;
+  gap: bpk-spacing-md();
+
+  &--dynamic {
+    cursor: pointer;
+  }
+
+  &-unselected {
+    background-color: $bpk-text-on-dark-day;
+    color: $bpk-text-on-light-day;
+  }
+
+  &-previous_selected {
+    background-color: $bpk-private-map-previous-selection-day;
+    color: $bpk-text-on-light-day;
+  }
+
+  &-selected {
+    background-color: $bpk-core-primary-day;
+    color: $bpk-text-on-dark-day;
+  }
+
+  &__wrapper {
+    position: relative;
+    display: flex;
+    padding: 0; // Prevent default <button> styling.
+    flex-direction: column;
+    align-items: center;
+    border: none;
+    background: none;
+  }
+
+  &--icon {
+    fill: $bpk-text-on-dark-day;
+  }
```

Comment:
Do we need this as a separate style? It could be that we encorporate it into the above states already and make sure we set them correctly for all states to avoid any issues/overriding in the icon